pages/apim/3.x/event-native/apim-event-native-api-management-overview.adoc

```diff
@@ -0,0 +1,89 @@
+[[apim-event-native-api-management-overview]]
+= Gravitee Event-native API Management (BETA)
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_event_native_api_management_overview.html
+:page-folder: apim/event-native
+:page-layout: apim3x
+
+[label label-version]#New in version 3.19.0#
+
+== Event-native API Management via the new evented Gateway (BETA)
+
+Gravitee Platform release 3.19.0 has introduced the first version (BETA) of a new type of Gateway and API definition - an evented Gateway that enables event-native API management.
+
+The new event-native policy execution engine is based on a modern and fully reactive architecture designed to address a number of challenges users have been facing with the existing v3 engine, and to build the foundation of Gravitee's event-native platform by adding features such as native support for Pub/Sub (Publish-Subscribe) design and enabling policies at message level. The new API definition makes it possible for the Gateway to “know” what is necessary in order for it to handle asynchronous API use cases.
```

Comment:
you mention the fact that the new engine is the foundation of event native platform which is true, but then it cannot be also event native by itself.

Reply:
Agreed - I have reworded this section and others to reflect this